src/components/Spinner/Spinner.tsx

```diff
@@ -19,16 +19,17 @@ export const Circle = styled(AnimatedView)`
 `;
 
 /**
- * @example ../docs/Spinner.md
+ * @example ./Spinner.md
  */
-const Spinner = () => {
+const Spinner: React.FC & { themePath: string } = () => {
   const rotateValue = useRef(new Animated.Value(0));
 
   const loop = Animated.loop(
     Animated.timing(rotateValue.current, {
       duration: 800,
       easing: Easing.linear,
       toValue: 1,
+      useNativeDriver: false,
```

Comment:
doesn't work with native driver?

Reply:
Looks like it does, moving it to true in the next branch of the conversion.